packages/api/src/utils/billing.js

```diff
@@ -1,9 +1,19 @@
 /* eslint-disable no-void */
 
+import { InvalidTosAgreementError, TosAgreementRequiredError } from 'src/errors'
+
 /**
  * @typedef {import('./billing-types').StoragePriceName} StoragePriceName
  */
 
+/**
+ * @param {import('./billing-types').Agreement} agreement
+ * @returns {boolean}
+ */
+function isAgreement (agreement) {
+  return agreement === 'web3.storage-tos-v1'
```

Comment:
right now this is fine because we aren't storing the ToS in the DB (need to have a future conversation about this cc @gobengo ).